src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -123,13 +125,13 @@ class DeBruijnGraph(val kmerSize: Int,
   }
 
   /**
-   *  Searches forward or backward from a node to find those connected by a unique path
+    *  Searches forward or backward from a node to find those connected by a unique path
```

Comment:
"ugh" because I had just convinced myself that first-asterisk was what we should standardize on, and then that says it should be second asterisk. cf #469, lmk which you want to do.